airgun/widgets.py

```diff
@@ -2100,7 +2100,20 @@ class AutoCompleteTextInput(TextInput):
     force the auto-completion list to be hidden.
```

Comment:
Since docstring here explains why we need this subclass in the first place, I think it would be good idea to add this new use-case (calling custom `clear()`) here as well.